docs/configurations.md

```diff
@@ -0,0 +1,216 @@
+# ScalarDB Configurations
+
+This page describes the available configurations for ScalarDB.
+
+## Transaction managers
+
+Implemented within ScalarDB are two transaction managers: Consensus Commit and gRPC. You can specify one of the transaction managers by using the `scalar.db.transaction_manager` property and then specify the configurations for the transaction manager.
```

Comment:
Although we can set `grpc` in `scalar.db.transaction_manager`,
I think it's pretty misleading to say we have two transaction managers because gRPC is not a transaction manager.
(More correctly, I feel Consens Commit is a transaction protocol, not a transaction manager.)

How about saying something like this? > @josh-wong @brfrn169 
(Maybe the sentences are broken up or we can reorganize the sections.)


ScalarDB provides its own transaction protocol called Consens Commit. You can use the Consensus Commit protocol directly through the ScalarDB client library or through a gRPC-based ScalarDB Server, which is a daemon process that manages ScalarDB transactions. 
When you use the Consensus Commit protocol directly through the ScalarDB client library, you specify `scalar.db.transaction_manager=consensus-commit` or you can omit the configuration since the `consensus-commit` is the default value.
When you use the Consensus Commit protocol through a gRPC-based ScalarDB Server, you need to specify `scalar.db.transaction_manager=grpc`.

Reply:
Sounds good! Thank you.

Reply:
@feeblefakie Thank you for the explanation!

I added a sentence about specifying `scalar.db.transaction_manager=grpc` to the `## ScalarDB Server (gRPC)` section (a sentence about specifying `scalar.db.transaction_manager=consensus-commit` already exists in the `## Consensus Commit` section), so I've simplified this paragraph with the first couple of sentences that you provided.

Since you feel that Consensus Commit is a transaction protocol rather than a transaction manager, what do you think about changing the heading for this section from `## Transaction managers` to `## Transaction protocols`? If so, I think we should also change mentions of `Consensus Commit transaction manager` to `Consensus Commit transaction protocol` throughout this doc.

Reply:
Yeah, I think the title should be revised. I also feel the overall structure might need to be reorganized since the second level titles are a bit inconsistent. (Transaction Managers, ScalarDB Server configurations, and Other ScalarDB configurations)

How about this? > @josh-wong @brfrn169 

# ScalarDB Configurations
## ScalarDB client configurations
### Use Consensus Commit directly
### Use Consensus Commit through ScalarDB Server
## ScalarDB Server configurations
## Other ScalarDB configurations

Reply:
@feeblefakie That structure makes sense to me. I've made revised the headings in https://github.com/scalar-labs/scalardb/commit/f2cf495154bdd0ed4a2fba47c35fd8cd1fe4d117.